package.json

```diff
@@ -36,9 +38,10 @@
     "./immediate.js": "./immediate-browser.js"
   },
   "scripts": {
-    "test": "standard && node test.js | faucet",
+    "test": "standard && nyc --silent node test.js | faucet",
```

Comment:
FYI @ralphtheninja `--silent` disables the coverage summary at the end of test output

Reply:
I think we should enable it and remove faucet.

Reply:
I'm okay with either. On the one hand, faucet reduces the output, on the other, the coverage summary is kinda nice (though when coverage matters, I prefer using `nyc report --reporter=html`).

Reply:
@ralphtheninja your call

Reply:
It would be optimal if we could have both `faucet` and the coverage report (maybe we can figure out a solution for this somehow?).

I just think it's really useful to have that report while testing locally. "Did I fuck up coverage? Did coverage change?"

Reply:
There's https://github.com/MegaArman/tap-nyc but it goes too far in summarizing IMO

Reply:
Maye we can just hack `faucet` to let stuff pass through. I'm guessing it's `faucet` and prevents the `nyc` output?

Reply:
`level-faucet` :)

Reply:
Probably difficult because `tap-parser` considers the test to be complete after

```
1..735     
# tests 735
# pass  735
           
# ok       
```

Perhaps when `tap-parser` emits "complete", we can switch stdin to pipe to stdout.